eth-contracts/contracts/ForceBridge.sol

```diff
@@ -315,6 +316,7 @@ contract ForceBridge {
         bytes memory recipientLockscript,
         bytes memory sudtExtraData
     ) public {
+        require (amount > 0);
```

Comment:
require(amount > 0, "lock amount should be greater than 0");

Reply:
when amount = 0, tx will be failed